src/components/edit-on-github-link/style.module.css

```diff
@@ -0,0 +1,17 @@
+.editLink {
```

Comment:
Ah, great callout re: font-weight. Looks like the Figma also now has no underline, so will make those tweaks 👍

TLDR; on `inline-flex` is cause then the link behaves like other text would, and we can `text-align` it. This only came up cause of previous positioning concerns when I was experimenting with placement in the `<Sidecar />` - it allowed the whole `<EditOnGithubLink />` stick to the right of the `<Sidecar />` like everything else because of the existing `text-align` setting. The `flex-end` was actually a holdover from having `display: flex` on there and trying to come up with an alternate solution for that right alignment (removed now!).

I dig the `display: flex` & `width: fit-content` - seems like a great alternative since we might not need the link to obey `text-align` if it's at the bottom of the page. I only recently learned about `fit-content` so that's why it didn't pop in my mind first! ☺️

Reply:
Thank you for elaborating! I super appreciate the context and totally makes sense. 😊